<launch>

  <node name="main_com"       pkg="meisekisisui"     type="main_com"       output="screen"  />

  <node name="important"      pkg="rosserial_server" type="socket_node"                     />

  <node name="realsense_info" pkg="realsense_info"   type="realsense_info">

  <node name="python"         pkg="rosserial_python" type="message_info_service.py"         />

  <arg name="P0" default="ACM0"/>
  <arg name="P1" default="ACM1"/>
  <arg name="P2" default="ACM2"/>
  <arg name="P3" default="ACM3"/>
  <arg name="P4" default="ACM4"/>
  <arg name="P5" default="ACM5"/>

  <node name="rosserial_tty$(arg P0)"  pkg="rosserial_server" type="serial_node" output="screen">
      <param name="port" value="/dev/tty$(arg P0)"/>
      <param name="baud" value="115200"/>
  </node>

  <node name="rosserial_tty$(arg P1)"  pkg="rosserial_server" type="serial_node" output="screen">
      <param name="port" value="/dev/tty$(arg P1)"/>
      <param name="baud" value="115200"/>
  </node>

  <node name="rosserial_tty$(arg P2)"  pkg="rosserial_server" type="serial_node" output="screen">
      <param name="port" value="/dev/tty$(arg P2)"/>
      <param name="baud" value="115200"/>
  </node>

  <node name="rosserial_tty$(arg P3)"  pkg="rosserial_server" type="serial_node" output="screen">
      <param name="port" value="/dev/tty$(arg P3)"/>
      <param name="baud" value="115200"/>
  </node>

  <node name="rosserial_tty$(arg P4)"  pkg="rosserial_server" type="serial_node" output="screen">
      <param name="port" value="/dev/tty$(arg P4)"/>
      <param name="baud" value="115200"/>
  </node>

  <node name="rosserial_tty$(arg P5)"  pkg="rosserial_server" type="serial_node" output="screen">
      <param name="port" value="/dev/tty$(arg P5)"/>
      <param name="baud" value="115200"/>
  </node>

</launch>

